plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationContext.java

```diff
@@ -41,12 +42,17 @@
 
     public ConfigurationContext(ConfiguratorRegistry registry) {
         this.registry = registry;
-        String prop = Util.fixEmptyAndTrim(System.getProperty(
-            CASC_YAML_MAX_ALIASES_PROPERTY,
-            System.getenv(CASC_YAML_MAX_ALIASES_ENV)
-        ));
+        String prop = getPropertyOrEnv(CASC_YAML_MAX_ALIASES_ENV);
         yamlMaxAliasesForCollections = NumberUtils.toInt(prop, 50);
         secretSourceResolver = new SecretSourceResolver(this);
+        mergeStrategy = getPropertyOrEnv("CASC_MERGE_STRATEGY");
```

Comment:
this is using the max aliases property, and setting it to the value of merge strategy, see line 53

Reply:
Great catch!